include/swift/Basic/InlineBitfield.h

```diff
@@ -82,6 +82,79 @@ namespace swift {
 
 #define SWIFT_INLINE_BITS(T) T##Bitfield T##Bits
 
+// XXX/HACK: templated max() doesn't seem to work in a bitfield size context.
+constexpr unsigned bitmax(unsigned a, unsigned b) {
+  return a > b ? a : b;
+}
+
+constexpr unsigned countBitsUsed(uint64_t arg) {
```

Comment:
LLVM has these in MathExtras.h, albeit not marked constexpr. It would be nice™ if we could reuse that?

Reply:
I agree that it would be nice™ to if we could reuse that. I can certainly try, but based on my experience with `max()` and this pull request, I suspect that clang doesn't like templated functions in the bit field size context.